### PR TITLE
Remove artifacts of database setup for GHSL rasters

### PR DESCRIPTION
**Pull request by matthiasschaub**
### Description
Remove artifacts of database setup for GHSL rasters.

### Corresponding issue
- None

### New or changed dependencies
- None

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
~- [ ] I have commented my code~
~- [ ] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)~
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

File: database/scripts/import/insert_srid_54009.sql [hunk 1]
```diff
@@ -1,14 +0,0 @@
-/* Required for GHS_SMOD raster */
```
Comment (Hagellach37) [inline database/scripts/import/insert_srid_54009.sql:1]: not sure if we might still need this in postgres? How do we query the ghspop or ghssmod rasters with rasterstats? Do we need to tranform the region geometry into 54009 first?

**Review by Hagellach37 — APPROVED** [review]
Hey,
looks good. Could you check if we still need 54009 as a spatial reference system?

File: database/scripts/import/insert_srid_54009.sql (hunk 1 above)
Comment (matthiasschaub) [reply database/scripts/import/insert_srid_54009.sql:1]: We do not need it at the moment. 
The OQT raster client (`rasterstats`) will transfer geometries using `pyproj` if necessary (no database involved in this step).

**Review by Hagellach37 — APPROVED** [review]

**Comment by Hagellach37** [comment]
should be ready now.

**Pull request merged**